the buildkite/g fail look not related to the patch , it fail at 

 FAILED: tools/flang/test/CMakeFiles/check-flang 
_bk;t=1709146033248 cmd.exe /C "cd /D C:\ws\src\build\tools\flang\test && C:\Python39\python.exe C:/ws/src/build/./bin/llvm-lit.py -v --xunit-xml-output C:/ws/src/build/test-results.xml --timeout=1200 --time-tests --param flang_site_config=C:/ws/src/build/tools/flang/test/lit.site.cfg.py C:/ws/src/build/tools/flang/test"
_bk;t=1709146033248 ninja: build stopped: subcommand failed.

--- a/llvm/docs/CommandGuide/llvm-readobj.rst
+++ b/llvm/docs/CommandGuide/llvm-readobj.rst
@@ -63,8 +63,8 @@ file formats.
 
 .. option:: --demangle, -C
 
- Display demangled symbol names in the output; the option only is for ELF and
- XCOFF file format.
+ Display demangled symbol names in the output. This option is only for ELF and
+ XCOFF file formats.
 
 .. option:: --expand-relocs
 

--- a/llvm/docs/CommandGuide/llvm-readobj.rst
+++ b/llvm/docs/CommandGuide/llvm-readobj.rst
@@ -101,8 +101,8 @@ file formats.
 
 .. option:: --no-demangle
 
- Not demangle symbol names in the output; the option is only for ELF and
- XCOFF file format. The option is enabled by default.
+ Do not demangle symbol names in the output. This option is only for ELF and
+ XCOFF file formats. The option is enabled by default.
 
 .. option:: --relocations, --relocs, -r
 

--- a/llvm/docs/CommandGuide/llvm-readobj.rst
+++ b/llvm/docs/CommandGuide/llvm-readobj.rst
@@ -185,10 +185,6 @@ The following options are implemented only for the ELF file format.
 
  Requires :option:`--bb-addr-map` to have an effect.
 
-.. option:: --demangle, -C
-
- Display demangled symbol names in the output.
-
 .. option:: --dependent-libraries
 
  Display the dependent libraries section.

